napari/utils/perf/_config.py

```diff
@@ -17,11 +17,11 @@
 class PerfmonConfigError(Exception):
     """Error parsing or interpreting config file."""
 
-    def __init__(self, message) -> None:
+    def __init__(self, message: str) -> None:
         self.message = message
 
 
-def _patch_perf_timer(parent, callable_name: str, label: str) -> None:
+def _patch_perf_timer(parent: Any, callable_name: str, label: str) -> None:
```

Comment:
```suggestion
from types import ModuleType

def _patch_perf_timer(parent: Union[ModuleType, type], callable_name: str, label: str) -> None:
```
Of course, import should be moved on top.

Reply:
👍 done